packages/genesys-spark-components/src/components/stable/gux-column-manager/gux-column-manager-item/gux-column-manager-item.scss

```diff
@@ -44,7 +52,10 @@
     flex: 0 1 auto;
     align-self: auto;
     order: 0;
-    color: #596373; //TODO https://inindca.atlassian.net/browse/COMUI-2314
+    color: var(
+      --gse-semantic-foreground-formControl-clickInput-enabled
+    ); //TODO https://inindca.atlassian.net/browse/COMUI-2314
```

Comment:
I noticed https://inindca.atlassian.net/browse/COMUI-2314 is in a Done state. Do we still need these TODOs here?